etc/kayobe/kolla/config/prometheus/rabbitmq.rules

```diff
@@ -5,69 +5,76 @@
 groups:
 - name: rabbitmq.rules
   rules:
-  - alert: RabbitmqNodeDown
-    expr: sum(rabbitmq_build_info{host_name!=""}) < 3
+  - alert: RabbitMQNodeDown
+    expr: sum(rabbitmq_build_info{instance!=""}) < 3
     for: 30m
     labels:
       severity: critical
     annotations:
-      description: Rabbitmq node down on {{ $labels.host_name }}
-  - alert: RabbitmqConsumersLowUtilization
+      description: RabbitMQ node down on {{ $labels.instance }}
+  - alert: RabbitMQConsumersLowUtilization
```

Comment:
I would rather we re-sync with the upstream repo here, but that feels like a nit:
https://awesome-prometheus-alerts.grep.to/rules#rabbitmq-1

Reply:
I will try to push some improvements upstream first and then synchronise. I will merge this in the meantime.